### PR TITLE
Handles the case when /sys/class/udc is empty

### PR DESCRIPTION
**Pull request by nascs**

### CODE REVIEW

File: root/lib/radxa-otgutils/radxa-otgutils.sh [hunk 1]
```diff
@@ -70,6 +70,10 @@ start()
     esac
 
     local UDC=$(ls /sys/class/udc/ | head -1)
+    if [ -z "$UDC" ] ; then
+        echo "UDC is NULL"
```
Comment (RadxaYuntian) [inline root/lib/radxa-otgutils/radxa-otgutils.sh]: `No USB Device Controller was detected. Did you set your OTG port to host mode?`

File: root/lib/radxa-otgutils/radxa-otgutils.sh [hunk 2]
```diff
@@ -70,6 +70,10 @@ start()
...
+        exit
```
Comment (RadxaYuntian) [inline root/lib/radxa-otgutils/radxa-otgutils.sh]: `exit 1`

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]

File: root/lib/radxa-otgutils/radxa-otgutils.sh [hunk 3]
```diff
@@ -70,6 +70,10 @@ start()
     esac
 
     local UDC=$(ls /sys/class/udc/ | head -1)
+    if [ -z "$UDC" ] ; then
```
Comment (RadxaYuntian) [inline root/lib/radxa-otgutils/radxa-otgutils.sh]: Also please follow existing code style.

File: root/lib/radxa-otgutils/radxa-otgutils.sh (hunk 2 above)
Comment (nascs) [reply root/lib/radxa-otgutils/radxa-otgutils.sh]: exit 1 的话， 执行 sudo systemctl start radxa-adbd.service 这条命令依然报错,  radxa-adbd.service 仍然处于 failed 状态, exit  的话， 执行 sudo systemctl start radxa-adbd.service 这条命令不会报错， radxa-adbd.service 处于active状态

**Review by RadxaYuntian — APPROVED** [review]

**Pull request merged**